eventuals/event-loop.h

```diff
@@ -781,9 +781,15 @@ class EventLoop final : public Scheduler {
               if (!completed_) {
                 CHECK(!started_);
                 started_ = true;
-
+#if !defined(_WIN32)
                 error_ = uv_poll_init(loop_, poll(), fd_);
-
+#else
+                if (GetFileType(fd_) == FILE_TYPE_PIPE) {
```

Comment:
1. GetFileType requires #include <fileapi.h>
2. GetFileType expects HANDLE as a parameter and not a random integer.